packages/storybook/.storybook/manager-head.html

```diff
@@ -1,4 +1,39 @@
 <!--
   Override the default favicon used in the Storybook in the browser tab.
 -->
- <link rel="icon" href="favicon.ico" sizes="any">
+<link rel="icon" href="favicon.ico" sizes="any">
+<style>
+    /* 
+       Storybook 10's toolbar icons (specifically the addon icons on the right) 
+       sometimes default to a dark grey (fgColor.default) instead of using barTextColor.
+       Since our toolbar background is dark green (Brand85), we force these icons to be white.
+       See https://github.com/storybookjs/storybook/issues/32789
```

Comment:
trigger a rebuild (current one seems stuck)
```suggestion
       See: https://github.com/storybookjs/storybook/issues/32789
```